cocos/2d/CCLabel.cpp

```diff
@@ -130,13 +131,18 @@ class LabelLetter : public Sprite
             return;
         }
 
-        Color4B color4(_displayedColor.r, _displayedColor.g, _displayedColor.b, _displayedOpacity);
+        auto displayedOpacity = _displayedOpacity;
+        if(!_letterVisible)
```

Comment:
I want LabelLetter not be rendered directly, it is desired to maintain the visible property in sprite equal to `false`.